packages/modules/order/src/utils/transform-order.ts

```diff
@@ -33,6 +33,11 @@ export function formatOrder(
     }
 
     mainOrder.items = mainOrder.items?.map((orderItem) => {
+      const isFormatted = isDefined(orderItem.detail?.fulfilled_quantity)
```

Comment:
question: When can this happen?

Reply:
I was playing a bit internally with formatting and added a few checks to skip when something is already formatted. At the end this is not being used, but I believe it doesn't hurt to keep the check in case the function is called more than once to format the Order.